src/resolve/resolved-dns-stream.c

```diff
@@ -386,6 +422,7 @@ int dns_stream_new(Manager *m, DnsStream **ret, DnsProtocol protocol, int fd) {
 
         s->n_ref = 1;
         s->fd = -1;
+        s->tfo_salen = 0;
```

Comment:
no need to initialize this, as we use new0() above, which initializes this anyway. The only fields that need explicit initialization are the ones those different from zero